src/features/session_replay/aggregate/index.js

```diff
@@ -433,6 +457,13 @@ export class Aggregate extends AggregateBase {
     return this.estimateCompression(this.payloadBytesEstimation + newBytes) + 1000
   }
 
+  actuallyStop (forceHarvest) {
```

Comment:
```suggestion
  /**
   * Forces the agent into OFF mode so that changing tabs or navigating
   * does not restart the recording. This is used when the customer calls
   * the stopRecording API.
   */
  forceStop (forceHarvest) {
```

Also curious why the contents of this function where not just in the event handler above. Maybe just to keep the event handlers clean and short, which is good.